### PR TITLE
Readme game instructions

### PR DESCRIPTION
**Pull request by iliebabcenco**
In this pull request, we want to merge a readme file that contains the description of the project and how to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Built With
+
+- Ruby
+
+## Getting Started
+
+In order to play or get a local copy you can clone this repository by using:
+
+git clone git@github.com:iliebabcenco/tic-tac-toe.git
+
```
Comment (elisha2kyakpo1) [inline README.md]: This area is missing information about how to have a copy, after someone clone the project, what next?

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+After cloning the repository, to play the game type ruby bin/main.rb in the terminal.
```
Comment (Dipeshtwis) [inline README.md]: [MANDATORY]

- [x] Missing to tell the user to go to the repo directory or else ruby bin/main.rb won't work. You can add more specific instructions. Think of first-time GitHub users with no technical knowledge.

For example,
`
  cd tic-tac-toe
`

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status: ♻️ Changes Required ♻️
## Hi @MarkoNS1990  and @iliebabcenco, I'm Dipesh your reviewer for this PR 👩‍🚀 

Hi Team, good work so far 🙌! I also tested your work and I really like it. However, for this Milestone, it is required some changes to fully meet the requirements.

- [x] Remember to create your README file thinking about new users without any knowledge of how a terminal works and other technical stuff you add to your README. Try explaining it with more clarity and specificity.

- To be able to make your README look fully professional is not just to make great documentation of your project, but also to write it with great grammar and make it descriptive.

See lower to see requested changes.

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional question).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

**Review by elisha2kyakpo1 — APPROVED** [review]
### Congratulations my fellow coders! I'm happy to see the requested changes being implemented.
[OPTIONAL] you can use backticks or greater than symbol for the code commands like:
> cd <your folder> this will make the file more readable.

**Pull request merged**